tests/view/crudapi/test_crud_api.py

```diff
@@ -9,24 +9,81 @@
 
 
 def test_get_found_object(app):
-    """ Tests for CrudeAPIView get method when the object exists. """
+    """
+    Tests for CrudeAPIView get method when the object exists.
+    FakeAPIView extends CrudeAPIView.
+    """
     with app.test_request_context(
-            "/test/crudapitest"
+        "/test/crudapitest"
     ):
         apiview = FakeAPIView()
         result = apiview.get(5)
     json_result = json.loads(result[0].get_data(as_text=True))
     assert result[0].is_json is True
     assert json_result == {"some_id": 5, "some_attr": "attr"}, \
-            "Wrong result returned from CrudeAPI view"
+        "Wrong result returned from CrudeAPI view"
     assert result[1] == HTTPStatus.OK, "Wrong response from CrudeAPI view"
 
 
 def test_get_not_found_object(app):
-    """ Tests for CrudeAPIView get method when the object does not exists. """
+    """
+    Tests for CrudeAPIView get method when the object does not exists.
+    FakeAPIView extends CrudeAPIView.
+    """
     with app.test_request_context(
-            "/api/crudapi"
+        "/api/crudapi"
     ):
         apiview = FakeAPIView()
         with pytest.raises(NotFound, message="Fake object not found"):
             apiview.get(0)
+
+
+def test_post_object(app):
+    """
+    Tests for CrudeAPIView post method.
+    FakeAPIView extends CrudeAPIView.
+    """
+    with app.test_request_context(
+        "/test/crudapitest"
+    ):
+        apiview = FakeAPIView()
+        payload = {"some_id": 6, "some_attr": "attr6"}
+        result = apiview.post(payload)
+    json_result = json.loads(result[0].get_data(as_text=True))
+    assert result[0].is_json is True
+    assert json_result == payload, "Wrong result returned from CrudeAPI view"
+    assert result[1] == HTTPStatus.OK, "Wrong response from CrudeAPI view"
+
+
+def test_put_object(app):
+    """
+    Tests for CrudeAPIView post method.
+    FakeAPIView extends CrudeAPIView.
+    """
+    with app.test_request_context(
+        "/test/crudapitest"
+    ):
+        apiview = FakeAPIView()
+        payload = {"some_id": 6, "some_attr": "attr6"}
+        result = apiview.put(payload)
+    json_result = json.loads(result[0].get_data(as_text=True))
+    assert result[0].is_json is True
+    assert json_result == payload, "Wrong result returned from CrudeAPI view"
+    assert result[1] == HTTPStatus.OK, "Wrong response from CrudeAPI view"
+
+
+def test_delete_object(app):
+    """
+    Tests for CrudeAPIView post method.
+    FakeAPIView extends CrudeAPIView.
+    """
+    with app.test_request_context(
+        "/test/crudapitest"
+    ):
+        apiview = FakeAPIView()
+        result = apiview.delete(object_id=5)
```

Comment:
@soin08 here you should also use the ``FAKE_OBJECT_ID`` constant, I'd say

Reply:
Fixed